
global ERRORS = "";

a = 5;
function fn1( x ) use( a )
{
	printlns( this, x, a );
}
fn1.apply( true, [] );

testEqual( ERRORS, "", "no errors on closure call(this)" );
